charts/loki/templates/_helpers.tpl

```diff
@@ -73,3 +73,13 @@ extensions/v1
 {{- end }}
 {{- end -}}
 
+
+{{/*
+Common labels
+*/}}
+{{- define "loki.labels" -}}
+app: {{ template "loki.name" . }}
+chart: {{ template "loki.chart" . }}
+release: {{ .Release.Name }}
+heritage: {{ .Release.Service }}
```

Comment:
there's some discrepancy here with the label names we use across charts and should probably try to be consistent. For example, in `loki-simple-scalable` and `loki-distributed` we us `helm.sh/chart` instead of just `helm`, and prefix a lot of other labels with `app.kubernetes.io`.

Reply:
I do realize this was just a refactor to reduce duplication, but it made be aware of this at the same time.

Reply:
in case we change it, it will require redeploying the whole cluster to apply the changes, so I left it unchanged. 
**let's do it in a separate pr in future... there are many things to be done before our charts become identical...
what do you think?**

I struggle more with the structure of values files in these 3 charts.  `loki`, `loki-distributed` and `loki-simple-scalable` use different ways and different sets of properties/values to create Loki's config ;)

Reply:
:+1: to doing in a separate PR